mucommander-translator/src/main/resources/dictionary.properties

```diff
@@ -737,6 +737,11 @@ shortcuts_dialog.quick_search.remove_last_char = Remove last character from quic
 shortcuts_dialog.quick_search.jump_to_previous = Jump to previous quick search result
 shortcuts_dialog.quick_search.jump_to_next = Jump to next quick search result
 shortcuts_dialog.quick_search.mark_jump_next = Mark/Unmark current file and jump to next search result
+terminal.find.label = Terminal - Find
+terminal.line_down.label = Terminal - Line Down
+terminal.line_up.label = Terminal - Line Up
+terminal.page_down.label = Terminal - Page Down
+terminal.page_up.label = Terminal - Page Up
```

Comment:
maybe we should split each of these to the operation and "Terminal - {}" for displaying in the shortcuts dialog, that way we can reuse the translated string elsewhere

Reply:
Not sure about that, see my comments for above (JediTerm string).

Reply:
PR updated